Sources/SmileID/Classes/SelfieCapture/SelfieViewModel.swift

```diff
@@ -329,10 +329,23 @@ public class SelfieViewModel: ObservableObject, ARKitSmileDelegate {
                 jobStatusResponse = try await SmileID.api.getJobStatus(
                     request: jobStatusRequest
                 ).async()
+                do {
+                    try LocalStorage.moveToSubmittedJobs(jobId: self.jobId)
+                } catch {
+                    print("Error moving job to submitted directory: \(error)")
+                    self.error = error
+                }
                 DispatchQueue.main.async { self.processingState = .success }
             } catch {
+                if SmileID.allowOfflineMode {
+
+                } else {
```

Comment:
can you remove the empty branch

Reply:
I moved this logic to `LocalStorage` actually, forgot to clean this up